backend/src/tests/auth.test.js

```diff
@@ -0,0 +1,46 @@
+import { assert } from "chai";
+import { request } from "chai-http";
+import server from "../../server.js";
+
+describe("auth route tests", () => {
+  it("should login a user that exists", (done) => {
+    request
+      .execute(server)
+      .keepOpen()
+      .post("/api/v1")
+      .send({ username: "johndoe", password: "johndoe@admin" })
+      .end((error, res) => {
+        assert.equal(res.status, 200);
```

Comment:
Also before asserting for the positives, you need to check if(error) done(error)